x-pack/plugins/maps/public/classes/styles/vector/properties/dynamic_icon_property.test.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// eslint-disable-next-line max-classes-per-file
+import { shallow } from 'enzyme';
+
+jest.mock('ui/new_platform');
+jest.mock('../components/vector_style_editor', () => ({
+  VectorStyleEditor: () => {
+    return <div>mockVectorStyleEditor</div>;
+  },
+}));
+
+import { COLOR_MAP_TYPE, FIELD_ORIGIN, VECTOR_STYLES } from '../../../../../common/constants';
+
+// @ts-ignore
+import { DynamicIconProperty } from './dynamic_icon_property';
+
+import React from 'react';
+import { StyleMeta } from '../style_meta';
+
+const mockField = {
```

Comment:
All of these mocks are duplicated from https://github.com/elastic/kibana/blob/master/x-pack/plugins/maps/public/classes/styles/vector/properties/dynamic_color_property.test.js#L21. How about setting up some testing utils to avoid duplicating everything?